accepted/wasmtime-one-dot-oh.md

```diff
@@ -0,0 +1,464 @@
+  features are not backported to LTS releases.
+
+* LTS versions in theory could be dropped entirely. There aren't known users who
+  are specifically asking for LTS versions and existing production users of
+  Wasmtime are likely to stay with released versions to get improvements as they
+  come out. This proposal assumes, however, that there are developers we're not
+  actively hearing from who would benefit from such LTS versions and we'd like
+  to accomodate them. If in practice it turns out that there is very little
+  usage of LTS versions we can consider dropping support for them at a later
+  date when such a conclusion is made (probably another RFC).
+
+* As currently proposed Wasmtime is guaranteed to receive a new major version
+  every 4 weeks. It might be the case, though, that within those 4 weeks of
+  development that Wasmtime didn't actually have any API-breaking changes occur.
+  Theoretically this could allow for a minor release of the form 2.1.0, for
+  example, if 2.0.0 was the previous release. There are a number of downsides
+  with trying to do this, though, including:
+
+  * Wasmtime developers now have to keep track of breaking changes and determine
+    whether any landed in the 4-week release window.
+  * This makes Wasmtime's versioning less predictable.
+  * LTS is somewhat more murky in this scenario. If it still happens every 5th
+    release we'll have to keep track of what the 5th release is, and if both
+    2.0.0 and 2.5.0 are LTS releases then it's not clear why projects would use
+    2.0.0 as opposed to 2.6.0 if it were the current version (since everything
+    is semver-compatible).
+  * If the `wasmtime` Rust crate didn't have any breaking changes it doesn't
+    mean that the embedding APIs also didn't have any breaking changes (or vice
+    versa too).
+
+  By having a new major version every 4 weeks it's definitely a form of "least
+  common denominator" solution but it also provides a predictable versioning
+  number as well as a clear indication of what is LTS and what isn't. For these
+  reasons it's proposed here to use a new major version every 4 weeks instead of
+  gauging via breaking changes or not.
+
+# Open questions
+[open-questions]: #open-questions
+
+- What sort of automation is required for Wasmtime developers to keep up with
+  once-a-month releases? The process described here is relatively expansive and
+  can take quite some effort to keep up with. For example:
+
+  * Can we entirely automate releases?
+  * Could this automation also get connected to all the other embeddings and
+    repositories?
+  * How hard will it be to make a patch release with a bug fix?
+  * How hard will it be to remember what branches need backporting?
+
+- Are releases skipped/delayed when they correspond to work holidays? For
+  example if a release happens to coincide with Christmas should it be delayed?
+  Skipped? Perhaps with entirely automated releases this wouldn't matter!
```

Comment:
I think we might answer this in this RFC already. Specifically, I'd propose including language in the relevant section above stating that a release might happen a week early or late to avoid coinciding with public holidays, but that the next release will still happen at its expected date.